src/containers/PokeSearchContainer.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react'
+import { PokeSearcher } from '../components/pokemon/PokeSearch'
+import { PokeList } from '../components/pokemon/PokeList'
+import { getAllPokemon } from '../services/PokeApi'
+
+export const PokeContainer = () => {
+    const [ data , setData ] = useState(null)
+    const [ search, setSearch ] = useState('')
+    const [ loading, setLoading ] = useState(true)
+
+    useEffect( async () => {
+        const allPokeData = await getAllPokemon()
+        setData(allPokeData)
+        
+        const interval = setTimeout(() => {
```

Comment:
Here I don't think we need a `setTimeOut`. I think we can update `setLoading(false)` right after `setData(allPokeData)` and it should still work :)

Reply:
Ah, this was my solution to avoid having the loading spinner from 'flashing' on screen for a brief second before the api data loads.  I actually just noticed that it's set to `100` instead of `1000`.  This would force every load spinner to be on screen for at least 1 second.